mep/accounts/admin.py

```diff
@@ -255,7 +334,8 @@ class AccountAdmin(admin.ModelAdmin):
                      'address__location__name',
                      'address__location__country__name', 'persons__name')
     fields = ('persons', 'card')
-    inlines = [AccountAddressInline, SubscriptionInline, ReimbursementInline]
+    inlines = [AccountAddressInline, SubscriptionInline, PurchaseInline,
```

Comment:
It doesn't make sense to me to include purchases here, since borrows aren't included - an account with lots of purchases could make this form slow and overwhelming. If we do decide to keep it, I think it should be listed last.

Reply:
I was seeing `Purchase` as analogous to `Reimbursement`,  since they're very samey, but I see they're closer to `Borrow` based on this and our brief discussion yesterday. Happy to pull it out.